core-ui/src/shared/ResourceForm/fields/K8sNameField.js

```diff
@@ -63,7 +66,7 @@ export function K8sNameField({
                 />
               </Tooltip>
             </div>
-            {showHelp ? (
+            {showHelp && inputInfo === undefined ? (
```

Comment:
`inputInfo !== undefined`

Reply:
And here you are checking `inputInfo` but not using it

Reply:
FormInput displays the description if it exists, so if we would use it here, it would be displayed twice. So here we just check if we should show or hide the default description.

Reply:
my bad, so it should be added in `gateways.yaml` in Port Name